lang/cs/Org.Apache.REEF.Utilities/Logging/Logger.cs

```diff
@@ -100,11 +116,6 @@ public static List<TraceListener> TraceListeners
             }
         }
 
-        public static void SetCustomLevel(Level customLevel)
```

Comment:
This is a public method in public class, so there could exist code outside of REEF codebase which uses it. Thus, we can't just remove it, and have to go through deprecation process: mark it as deprecated in this release (add `[Obsolete("Deprecated in 0.15. Will be removed, please use Logger.GlobalLevel instead"`), and once 0.15 is released, remove the method as part of 0.16